examples/webgl_modifier_subdivision.html

```diff
@@ -173,22 +173,28 @@
 				 	'<br><br>Subdivisions: '  + subdivisions +
 					' <a href="#" onclick="nextSubdivision(1); return false;">more</a>/<a href="#" onclick="nextSubdivision(-1); return false;">less</a>' +
 					'<br>Geometry: ' + dropdown + ' <a href="#" onclick="nextGeometry();return false;">next</a>' +
-					'<br><br>Vertices count: before ' + geometry.vertices.length + ' after ' + smooth.vertices.length +
-					'<br>Face count: before ' + geometry.faces.length + ' after ' + smooth.faces.length
+					'<br><br>Vertices count: before ' + geometry.vertices.length + ' after ' + (smooth.attributes.position.array.length/3) +
+					'<br>Face count: before ' + geometry.faces.length + ' after ' + (smooth.attributes.position.array.length / 3)
+                /* since the result is unindex'd the face count will always be directly related to the vertice count */
 				; //+ params.args;
 			}
 
 			function addStuff() {
 
-				if ( cube ) {
-
+				if ( cube != false ) {
```

Comment:
Isn't cube an `object3D`? Do you mean `undefined`? I think the code you inherited had unusual variable names. : - )